server/src/main/kotlin/org/kryptonmc/krypton/packet/out/play/PacketOutSetPassengers.kt

```diff
@@ -28,19 +28,16 @@ import org.kryptonmc.krypton.util.writeVarInt
 @JvmRecord
 data class PacketOutSetPassengers(
     override val entityId: Int,
-    val count:  Int,
     val passengers: List<Entity>,
 ) : EntityPacket {
 
     constructor(entity: KryptonEntity) : this(
         entity.id,
-        entity.passengers.size,
         entity.passengers
     )
 
     override fun write(buf: ByteBuf) {
         buf.writeVarInt(entityId)
-        buf.writeVarInt(count)
         buf.writeIntArray(passengers.map { (it as KryptonEntity).id }.toIntArray())
```

Comment:
All I can say to this is:
![No](https://c.tenor.com/F4vgEqTjG7AAAAAd/tracey-morgan-no.gif)

You realise this is going to do the following:
- Create a new list of integers
- Create a new integer array

I can recommend the following suitable replacement:
```suggestion
        buf.writeVarInt(passengers.size)
        passengers.forEach { buf.writeVarInt((it as KryptonEntity).id) }
```

Reply:
Ah yes. That seems like a much more efficient method